### ISSUE TITLE
Vertical toolbar needs to be themed

### ISSUE THREAD

**Issue by craxal** [microsoft/AzureStorageExplorer#4637]
Currently, the colors for the vertical toolbar are hardcoded. This does not work well for high contrast (app or system) situations.

Specifically, if system high contrast is turned on and set to the default White, the toolbar icons are not visible.

![image](https://user-images.githubusercontent.com/3473026/123880552-c515e400-d8f7-11eb-8abf-0a44de428b34.png)
